.rubocop_ruby.yml

```diff
@@ -70,10 +70,6 @@ AllCops:
   # Otherwise we fallback to the oldest officially supported Ruby version (2.0).
   TargetRubyVersion: 2.7
 
-  RSpec:
-    Patterns:
-      - "(?:^|/)spec/"
-      - "(?:^|/)test/"
```

Comment:
do you want to remove it?

Reply:
yes, it is causing rubocop errors and does not add anything